openpype/lib/openpype_version.py

```diff
@@ -60,9 +61,16 @@ def is_running_from_build():
 def is_running_staging():
     """Currently used OpenPype is staging version.
 
+    Deprecated:
+        Since 3.15
+
     Returns:
         bool: True if openpype version containt 'staging'.
     """
+    warnings.warn(
```

Comment:
This function is used to determine what icon is used. We should be able somehow identify if staging is used or not...

Reply:
true but it does it currently from the version. Maybe it should check the staging version set in the settings, compare it with the one requested and if it matches use staging icon?

Reply:
But there is no way how we can know that user used `--use-staging` command.